AugurProject/augur
# Make no tradable

<https://github.com/AugurProject/augur/issues/3698>

expose NO for trading. cleaned up some logic. verify NO is tradable on YesNo and user gets long/short positions

noticed that user created market infos were not getting loaded.

to test:
create Yes No market with orders
when the market is created open market liquidity and create orders
when the market liquidity is created nav to market and create positions.
verify user can create positions on No and Yes outcomes.

![image](https://user-images.githubusercontent.com/3970376/65168756-c9b32600-da0a-11e9-9223-8f357829d985.png)


![image](https://user-images.githubusercontent.com/3970376/65168763-cf107080-da0a-11e9-9ffc-04ac44ed4a16.png)
